core/connector.go

```diff
@@ -48,24 +49,30 @@ type Connector interface {
 	AppName(connID string) (string, bool)
 	// LinkApp links the app and connection.
 	LinkApp(connID string, name string, observed []byte)
```

Comment:
share we use `sfn` instead of `App` after we introduced `Source` here ?

Reply:
Yeah, It's right.